index.md

```diff
@@ -3,18 +3,14 @@
 NASA Astrophysics is developing the [Fornax Initiative](https://pcos.gsfc.nasa.gov/Fornax/), a cloud-based system that
 brings together data, software, and computing so that researchers can focus on science.
 
-[Fornax Science Console](https://science-console.fornax.sciencecloud.nasa.gov/) | [Documentation](https://nasa-fornax.github.io/fornax-documentation/) | [Fornax Community Forum](https://discourse.fornax.sciencecloud.nasa.gov/)
+[Fornax Science Console](https://science-console.fornax.sciencecloud.nasa.gov/) | [Documentation](xref:fornax-docs) | [Fornax Community Forum](https://discourse.fornax.sciencecloud.nasa.gov/)
 
 ## Tutorial Notebooks
 
-```{toctree}
----
-maxdepth: 2
-caption: Tutorial Notebooks
----
+This collection of tutorial notebooks consists of fully worked science use cases for all users.
+Common goals of the notebooks are the usage of archival data from all NASA archives, cross-archive work, big data, and computationally intensive science.
+Currently, there are four major topics for which we have notebooks, use the navigation menu on the left hand side to explore them.
 
-forced_photometry/README
-light_curves/README
-spectroscopy/README
-crossmatch/README
-```
```

Comment:
I'd like to get feedback from others about where these TOCs show up. Personally, I think it's valuable to have them listed on the index page and the readme for each section.